examples/extending-objects/Abstract Classes/Go/person.go

```diff
@@ -0,0 +1,22 @@
+// Golang doesn't really have Abstract Classes, so this is an alternative way of implementing them
+package main
+
+import "fmt"
+
+type Person interface {
+	String() string
+	IncreaseAge()
+}
+
+type AbstractPerson struct {
```

Comment:
Add documentation here as if this were a class (similar to how you broke down the function documentation here: https://github.com/johnfraserss/ICS4U/pull/36/files#diff-4052028a1dfc29511be5c4f6e13333518fd18e1a5dbef8e3cb3e8c61e497d330 )

Reply:
Actually, this is a better example: https://github.com/johnfraserss/ICS4U/pull/36/files#diff-df8fdb95e9a748bd6204cb634d74c8537afba9d91159b413c5c202061c38c07d